proxy/coco/src/state.rs

```diff
@@ -967,87 +968,4 @@ mod test {
 
         Ok(())
     }
-
-    #[tokio::test]
-    async fn create_with_existing_remote_with_reset() -> Result<(), Box<dyn std::error::Error>> {
```

Comment:
Are we replacing these tests with something?

Reply:
I outlined why I removed them in the commit message https://github.com/radicle-dev/radicle-upstream/pull/1190/commits/ee3c24bf3232cd09edb31c4ce115c58352cd3596. Let me know if you have any questions about it :)